tutorials/measurement_chain.ipynb

```diff
@@ -0,0 +1,553 @@
+{
```

Comment:
<p>I'd like a concluding a small paragraph summarizing what we've just learned.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
Done